examples/with-nodejs/server-list-campaigns.ts

```diff
@@ -12,7 +12,6 @@ voucherify.campaigns
 		limit: 10,
```

Comment:
Let's leave .js files as they were and just do the same but with typescript or have separate directory `with-nodejs-typescript`